modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -932,6 +953,10 @@ Status description:: error: syntax error or access rule violation - cannot modif
 
 Status description:: error: syntax error or access rule violation - cannot delete own user. Cannot delete the user record of the current user.
 
+=== 42NA6
+
+Status description:: info: invalid alias target. Aliases are not allowed to target composite databases.
+
```

Comment:
This one is totally different between the codebase and the spreadsheet, not even about the same topic 😅 Do you know when it was changed and can verify which is correct @anderson4j and @WilcoNeo ?

Reply:
I did not change the code as far a I can see. 
To me it looks like this one is more correct than what is currently in the code.

Reply:
@Lojjs, @WilcoNeo, shall I merge this PR as is?

Reply:
Yes looking as the usage, it looks like the codebase is wrong and docs and spreadsheet is correct. I will make a card to update the codebase